src/LibraryManager.Vsix/Shared/LibraryCommandService.cs

```diff
@@ -144,10 +144,7 @@ public bool IsOperationInProgress
                 {
                     Dependencies dependencies = Dependencies.FromConfigFile(configFilePath);
                     Manifest manifest = await Manifest.FromFileAsync(configFilePath, dependencies, cancellationToken).ConfigureAwait(false);
-                    if (manifest != null)
-                    {
-                        manifests.Add(configFilePath, manifest);
-                    }
+                    manifests.Add(configFilePath, manifest);
```

Comment:
We now want to report errors when manifest is null. So callers need to check for null and write the proper errors,